Dangerfile

```diff
@@ -1,46 +1,30 @@
 # frozen_string_literal: true
 
-def wip_feature?
-  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
-  has_wip_title = github.pr_title.include?('WIP')
-
-  has_wip_label || has_wip_title
-end
+github.dismiss_out_of_range_messages
 
-# these are reviewers actively providing feedback and potentially changing the state of the PR (approved, changes-requested)
-def active_reviewers?
-  repo_name = github.pr_json['base']['repo']['full_name']
-  pr_number = github.pr_json['number']
+# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
+rubocop.lint(files: [], force_exclusion: true, inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
 
-  !github.api.pull_request_reviews(repo_name, pr_number).empty?
-end
+manifest_pr_checker.check_all_manifest_lock_updated
 
-# requested_teams / requested_reviewers are users initially requested to review a PR, who haven't reacted yet
-def requested_reviewers?
-  has_requested_reviews = !github.pr_json['requested_teams'].to_a.empty? || !github.pr_json['requested_reviewers'].to_a.empty?
-  has_requested_reviews || active_reviewers?
+# skip remaining checks if we're during the release process
+if github.pr_labels.include?('Releases')
+  message('This PR has the `Releases` label: some checks will be skipped.')
+  return
 end
 
-return if github.pr_labels.include?('Releases')
-
-github.dismiss_out_of_range_messages
-
-manifest_pr_checker.check_all_manifest_lock_updated
-
 labels_checker.check(
+  do_not_merge_labels: ['Do Not Merge'],
   required_labels: [//],
   required_labels_error: 'PR requires at least one label.'
 )
 
-view_changes_need_screenshots.view_changes_need_screenshots
+view_changes_checker.check
 
-pr_size_checker.check_diff_size
+pr_size_checker.check_diff_size(max_size: 500)
 
 milestone_checker.check_milestone_due_date(days_before_due: 2)
 
-# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
-rubocop.lint(files: [], inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
-
-warn('PR is classed as Work in Progress') if wip_feature?
+warn('PR is classed as Work in Progress') if github_utils.wip_feature?
 
-warn("No reviewers have been set for this PR yet. Please request a review from **@\u2028Automattic/pocket-casts-ios**.") unless requested_reviewers?
+warn("No reviewers have been set for this PR yet. Please request a review from **@\u2060Automattic/pocket-casts-ios**.") unless github_utils.requested_reviewers? || github.pr_draft?
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/a5d7b9d2d6e2133baad215aaf91ad2dc691dca8a/Dangerfile#L30"></span>Layout/LineLength: Line is too long. [181/180]</td>
    </tr>
  </tbody>
</table>